HW1/DubovskyiIhor/Task1.py

```diff
@@ -0,0 +1,10 @@
+print('What is your name?')
```

Comment:
concise version
age = input("Hello, How old are you?') 
print('Your age is ', age)
city = input('Where do you live?')  
print('You live in ', city)